server/mdm/apple/commander.go

```diff
@@ -171,6 +171,31 @@ func (svc *MDMAppleCommander) InstallEnterpriseApplicationWithEmbeddedManifest(
 	return svc.EnqueueCommand(ctx, hostUUIDs, string(raw))
 }
 
+func (svc *MDMAppleCommander) AccountConfiguration(ctx context.Context, hostUUIDs []string, uuid, fullName, userName string) error {
```

Comment:
Nit, but since this enqueues a command, should it be called `ConfigureAccount` instead? It executes an action.

Reply:
it's a tough one, the command is `AccountConfiguration` which sounds awkward, but so far we have been matching the method name with the command name. I don't have a strong opinion

Reply:
Ah yeah, this one's on Apple! :) I think it makes sense to have the same name as the command.